### ISSUE TITLE
The notifications of the  Pomodoro and Pomodoro  Break Timer is very confusing.

### ISSUE THREAD

**Issue by Qi2013** [toggl/toggldesktop#2533]
Hi guys, I like toggl. However, the Pomodoro timer is really confusing.

What has happened:
After the predefined period of time, the Pomodoro Timer will show a notification "you have spent ......" together with two buttons "continue" and "start new" 
![grafik](https://user-images.githubusercontent.com/3496689/43791355-b72a1816-9a75-11e8-92ff-1d4d82f8cdc7.png)
At the same time, a pomodoro break timer has been started in background without notifying the user!

What I expected:
After that period of time, a notification with ONE botton asking if I want to take a break or not. 
Only after I have clicked that YES button, another timer will start. 
After the break it will ask me if I want to continue or start a new task.

**Comment by IndrekV** [comment]
@Qi2013 
Currently there is different setting for pomodoro break. If it is not enabled no "break" entry is started when pomodoro timer runs out. This makes it possible for the user to decide when to start tracking again and avoids unnecessary "break" time entry. Does that answer your question or did you mean that you want the pomodoro break time entry and just want to control when it starts?
